package.json

```diff
@@ -74,9 +75,15 @@
     "lint-staged": "^10.0.2",
     "metro-react-native-babel-preset": "^0.59.0",
     "react-dom": "^16.8.3",
+    "react-native-svg-transformer": "^0.14.3",
     "standard-version": "^7.1.0"
   },
   "optionalDependencies": {
     "ios-deploy": "^1.9.4"
+  },
+  "rnpm": {
```

Comment:
Do we still need this here even with `react-native.config.js` 🤔 
The documentations appears to state otherwise: https://github.com/react-native-community/cli/blob/master/docs/configuration.md#apps

Reply:
No, we don't :D

Reply:
Yeah, but we're on v0.60+ anyway so we should be fine 👌